hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/ConvenientSourceP.java

```diff
@@ -109,6 +105,7 @@ public boolean isCooperative() {
     @Override
     protected void init(@Nonnull Context context) {
         // createFn is allowed to return null, we'll call `destroyFn` even for null `ctx`
+        ManagedContext managedContext = context.serializationService().getManagedContext();
```

Comment:
We should get the `MangedContext` from `context.jetInstance().hazelcastInstance()` downcasted to `HazelcastInstanceImpl`